storm-core/src/jvm/backtype/storm/security/auth/AuthUtils.java

```diff
@@ -276,4 +313,26 @@ public static String get(Configuration configuration, String section, String key
         }
         return null;
     }
+
+    private static final String USERNAME = "username";
+    private static final String PASSWORD = "password";
+
+    public static String makeDigestPayload(Configuration login_config, String config_section) {
```

Comment:
Technically this generates the secret payload, not the digest, but that is really minor.